### PR TITLE
Multi manifest support v2

### PR DESCRIPTION
**Pull request by jyasveer**

### CODE REVIEW

File: bayesian/api_v1.py [hunk 1]
```diff
@@ -339,85 +338,77 @@ def get(external_request_id):
...
-
-        if stack_result is not None and 'task_result' in stack_result:
-            if stack_result["task_result"] is not None:
-                if 'user_stack_info' in stack_result["task_result"]:
-                    stack_data = stack_result["task_result"]["user_stack_info"]
+        stacks = []
+        recommendations = []
 
+        if stack_result != None and 'task_result' in stack_result:
+            if stack_result["task_result"] != None:
```
Comment (miteshvp) [inline bayesian/api_v1.py]: started_at = stack_result.get("task_result", {}).get("_audit", {}).get("started_at")

**Comment by jyasveer** [comment]
I have done pull --rebase and push -f already.

File: bayesian/utils.py [hunk 2]
```diff
@@ -358,6 +358,12 @@ def retrieve_worker_result (rdb, external_request_id, worker):
         result = row.to_dict()
     return result
 
+def get_item_from_list_by_key_value(items, key, value):
+    for item in items:
+        if (item[key] == value):
```
Comment (jpopelka) [inline bayesian/utils.py]: redundant parenthesis

File: bayesian/api_v1.py [hunk 3]
```diff
@@ -339,102 +344,112 @@ def get(external_request_id):
...
 
-        if stack_result is not None and 'task_result' in stack_result:
-            if stack_result["task_result"] is not None:
-                if 'user_stack_info' in stack_result["task_result"]:
-                    stack_data = stack_result["task_result"]["user_stack_info"]
-
-                if '_audit' in stack_result["task_result"]:
-                    started_at = stack_result["task_result"]["_audit"]["started_at"]
-                    finished_at = stack_result["task_result"]["_audit"]["ended_at"]
+        if stack_result != None and 'task_result' in stack_result:
```
Comment (jpopelka) [inline bayesian/api_v1.py]: Use `is not None` instead of `!= None`

**Pull request merged**